### DISCUSSION TITLE
Is it possible to identity the object name with its semantic id?

### DISCUSSION THREAD

**Discussion by shengyin1224** [StanfordVL/BEHAVIOR-1K#1052] [Q&A] [answered]
Thanks for your wonderful work. I use codes to draw 2d bboxes of objects and get observation data which contains semantic id and its corresponding value. 

But I wonder whether it is possible to identify the exact object name of the semantic id or there are some other methods? If there are 3 straight chairs in the view, all of them will use the same semantic id.

**Comment by hang-yin** [comment, marked as answer]
Hi @shengyin1224 , absolutely! Please refer to the instance segmentation modalities detailed here: https://behavior.stanford.edu/omnigibson/modules/sensors.html#vision

Alongside the observation itself, when you call the get_obs method, you should be also getting a information dictionary. The ID to name mapping should be included in there. One other way to do this by calling the `semantic_class_id_to_name` or `semantic_class_name_to_id` in `omnigibson/utils/constants.py`
